cli/internal/codegen/testdata/expected_golang.go

```diff
@@ -371,10 +371,10 @@ func (c *svcClient) RequestWithAllInputTypes(ctx context.Context, params SvcAllI
 	// Copy the unmarshalled response body into our response struct
 	respDecoder := &serde{}
 
-	resp.A = respDecoder.ToTime("A", respHeaders.Get("x-alice"), false)
 	resp.B = respBody.B
 	resp.C = respBody.C
 	resp.Dave = respBody.Dave
+	resp.A = respDecoder.ToTime("A", respHeaders.Get("x-alice"), false)
```

Comment:
Why the ordering change? Is it due to the refactoring or is the output indeterministic?

Reply:
It's due to the refactoring. I can change it back to make it consistent